package.json

```diff
@@ -184,14 +182,6 @@
       "scripts/package.json"
     ],
     "versionGroups": [
-      {
```

Comment:
This is unneeded because test-bundles no longer uses a different webpack version